src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -117,6 +130,29 @@
     completeColumnDefs = [activityErrorColumnDef, ...(columnDefs ?? []), activityActionColumnDef];
   }
 
+  onMount(() => {
+    document.addEventListener(`keydown`, onKeyDown);
```

Comment:
This might be intentional and I'm still trying to think through it but having this keyboard listener be part of the activity directives table means that this global copy+paste for directives can only happen if the user has the table open. This is probably fine since when you click on an activity directive in the timeline the activity directive table auto opens if necessary which would initialize the listener. Only case is if you click on a directive, swap the directive table to something else, and try to copy paste. That's probably not enough of an edge case to care about though?